### DISCUSSION TITLE
How to perform that magic closing bracker trick?

### DISCUSSION THREAD

**Discussion by ghost** [Mair/esp32-course#128] [Course Qeustions]
Hi, Mair!( and all taking this course)
Just a short question, in lesson 
[wi-fi connect 5 error](https://learnesp32.com/videos/12/8_wi-fi%20connect%205%20-%20error)
at 5:30, while you were adding a pair of brackets, 
you somehow magically moved the positon of the end closing bracket! 
How did you do that?
Can you share me that awesome trick?

Zt.

**Comment by Mair** [comment]
Hi @ztliu 
I think what you are referring to is is the "end" key" or ctr right-arrow
